Kickstarter-iOS/Beta.entitlements

```diff
@@ -1,26 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-  <dict>
-    <key>aps-environment</key>
-    <string>production</string>
-    <key>com.apple.developer.associated-domains</key>
-    <array>
-      <string>applinks:www.kickstarter.com</string>
-      <string>applinks:native.dev.kickstarter.com</string>
-      <string>applinks:staging.kickstarter.com</string>
-      <string>applinks:click.e.kickstarter.com</string>
-      <string>applinks:click.em.kickstarter.com</string>
-      <string>applinks:email.kickstarter.com</string>
-      <string>applinks:emails.kickstarter.com</string>
-      <string>applinks:e2.kickstarter.com</string>
-      <string>applinks:e3.kickstarter.com</string>
-      <string>applinks:clicks.kickstarter.com</string>
-      <string>applinks:ea.kickstarter.com</string>
-      <string>applinks:me.kickstarter.com</string>
-      <string>webcredentials:kickstarter.com</string>
-    </array>
-    <key>com.apple.developer.ubiquity-kvstore-identifier</key>
-    <string>$(TeamIdentifierPrefix)$(CFBundleIdentifier)</string>
-  </dict>
+<dict>
+	<key>aps-environment</key>
+	<string>production</string>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:staging.kickstarter.com</string>
+		<string>webcredentials:kickstarter.com</string>
```

Comment:
Did you find out if it's possible for the beta app to also handle the same links as the production app? Or is it a 1:1 mapping only? It'd be nice to keep the environments equivalent (at least to prevent confusion when we're testing) but I also understand if that would make everything a complete and total mess. Or if Apple doesn't allow it.

Reply:
So we could definitely do this, but it would require updating our apple-app-site-association files to include the release app id and the beta app id. (If there are multiple app ids provided, Apple will open the first app it finds a match for or maybe the one we specify first; the documentation is lacking and I think it depends on how the file is structured.) I'm a little hesitant to do that because these files are all publicly available and adding references to our Beta app to prod-based files seems weird to me. (But the email links app site association files already include the Alpha app, so maybe it doesn't matter...) Happy to revisit this if you think it'd be helpful!

Reply:
Hmm yeah that's an interesting point! I'm not worried about it (I can't imagine anything you can do just by knowing the identifier of our beta app) - to pick one at random, it looks like Google includes enterprise and dev versions [in their site association](https://google.com/apple-app-site-association).

However, I also don't feel like fixing this is a huge priority - more of a nice-to-have so that the beta and release apps stay at parity. Maybe throw it in a low-priority ticket?

Reply:
(Mostly my thought on the ticket is - if someone from marketing comes to us and asks why it's broken, we can point at it and then maybe decide to prioritize it.)

Reply:
Good call; I created https://kickstarter.atlassian.net/browse/MBL-1107